build.gradle

```diff
@@ -40,8 +40,8 @@ test {
 dependencies {
     api 'com.squareup.retrofit2:retrofit:2.9.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
-    implementation 'com.squareup.okhttp3:okhttp:3.14.9'
-    implementation 'com.squareup.okio:okio:1.17.5'
+    implementation 'com.squareup.okhttp3:okhttp:4.12.0'
+    implementation 'com.squareup.okio:okio:3.9.1'
```

Comment:
I think these newer versions of okhttp and okio depend on the Kotlin standard library, which I would rather not have to include given this is a java client :)